packaging/snapd.mk

```diff
@@ -86,6 +86,7 @@ install:: snap | $(DESTDIR)$(bindir)
 	install -m 755 $^ $|
 
 # Install snapctl snapd, snap-{exec,update-ns,seccomp} into /usr/lib/snapd/
+# (but not snap-verify)
```

Comment:
There's a fragment for core systems (though we're not using this in Debian yet) that you could extend as well.